draft_talon_helpers.py

```diff
@@ -5,13 +5,40 @@
 
 mod = Module()
 ctx = Context()
+ctx.matches = r"""
```

Comment:
I'm not sure this addition is right. I made a Context so that I could have a tag applied even when the window isn't focussed. This lets you run draft submit while the window is open but you have the text area you want to paste into focussed.

With this addition, would the tag only be applied when the draft window has focus or something like that?

Reply:
This addition is for those actions that need to be overridden when the draft UI actually has focus. We don't want e.g.

`edit.selected_text()` 

overridden in this way when it's not focused since the implementation is specific to the text area, and the user may be doing something in another application.

Reply:
You're correct, however, this will break your tag. Good catch.